<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>auto drill</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="2,2,1">
    <item row="0" column="0" rowspan="2">
     <widget class="QWidget" name="boardDrillsWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="cameraWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout_zoom">
      <item row="1" column="0">
       <widget class="QSlider" name="verticalSlider_cameraZoom">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Zoom</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>58</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QTreeWidget" name="treeWidget_holes">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_trafo">
      <property name="title">
       <string>Transformation</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_addPoint">
         <property name="text">
          <string>Add Selected Point</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_removeAll">
         <property name="text">
          <string>Remove All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Please select 3 or 4 points.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Points used:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_trafoPoints">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>138</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_loadDrillFile"/>
    <addaction name="action_writeGCode"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>S&amp;ettings</string>
    </property>
    <addaction name="action_dialogDrills"/>
    <addaction name="action_dialogDrillParameter"/>
    <addaction name="action_dialogCamera"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_loadDrillFile">
   <property name="text">
    <string>&amp;Load Drill File</string>
   </property>
  </action>
  <action name="action_dialogDrills">
   <property name="text">
    <string>&amp;Drills</string>
   </property>
  </action>
  <action name="action_dialogCamera">
   <property name="text">
    <string>&amp;Camera Settings</string>
   </property>
  </action>
  <action name="action_writeGCode">
   <property name="text">
    <string>&amp;Write G-Code Files</string>
   </property>
  </action>
  <action name="action_dialogDrillParameter">
   <property name="text">
    <string>Drill Parameter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
